downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/CellularComponentAnnotationBuilder.java

```diff
@@ -35,7 +36,7 @@ public static void processCellularComponents(GKInstance reactionInst) throws Exc
                 if (!GOAGeneratorUtilities.isExcludedMicrobialSpecies(taxonIdentifier)) {
                     if (!speciesWithAlternateGOCompartment.contains(taxonIdentifier)) {
                         // For CC, we are looking at ALL proteins in a ReactionlikeEvent
-                        generateGOCellularCompartmentLine(proteinInst, reactionInst, taxonIdentifier);
+                        goaLines.add(generateGOCellularCompartmentLine(proteinInst, reactionInst, taxonIdentifier));
```

Comment:
A fair bit of the method logic is devoted to logging reasons to exempt/exclude proteins from being added to `goaLines`.  Creating some helper functions so the logic reads something like this might be clearer:

```
String reasonToExempt = validateProtein(proteinInst);
if (reasonToExempt.isEmpty()) {
    goaLines.add(...)
} else {
    logger.info(reasonToExempt);
}
```

Reply:
I'll defer this suggestion to a 'refactor' PR in the future

Reply:
Sure, maybe after merging this PR, could you create a branch for refactoring so it stays on the radar?